Components/alerts/alert.css

```diff
@@ -0,0 +1,92 @@
+:root {
+    --secondary-bg-color: #c3dbff;
+    --warning-bg-color: #fce6cf;
+    --error-bg-color: #ffd0d0;
+    --success-bg-color: #a8fcd6;
+}
```

Comment:
If this is common colour then you can include it in base CSS but if specific to only alert then fine.

Reply:
This is specific to alert. In the future, if it becomes common I shall move it. 👍🏽